pkg/detectors/dockerhub/dockerhub.go

```diff
@@ -79,8 +91,15 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 					// Valid credentials can still return a 401 status code if 2FA is enabled
 					if (res.StatusCode >= 200 && res.StatusCode < 300) || (res.StatusCode == 401 && strings.Contains(string(body), "login_2fa_token")) {
 						s1.Verified = true
+					} else if res.StatusCode == 401 {
+                                        // The secret is determinately not verified (nothing to do)
+                                	} else {
+						s1.VerificationError = fmt.Errorf("unexpected HTTP response status %d", res.StatusCode)
```

Comment:
This isn't actually possible any more - you need to use [a function](https://github.com/trufflesecurity/trufflehog/blob/main/pkg/detectors/detectors.go#L68) to set these errors now. You'll need to resolve this when you update your base branch.